dali/test/python/reader/test_numpy.py

```diff
@@ -658,13 +669,10 @@ def check_pad_last_sample(device):
             for _ in range(2):
                 pipe_out = pipe.run()
                 for i in range(batch_size):
-                    pipe_arr = to_array(pipe_out[0][i])
+                    out_arr = to_array(pipe_out[0][i])
+                    out_prop = to_array(pipe_out[1][i])
```

Comment:
SourceInfo has a first-class support, so you could just do:
```suggestion
                    out_prop = pipe_out[0][i].source_info()
```

Reply:
Fixed